ios/RNCalendarEvents.m

```diff
@@ -615,6 +663,19 @@ - (NSDictionary *)serializeCalendarEvent:(EKEvent *)event
                 else {
                     [formattedAttendee setValue:@"" forKey:@"name"];
                 }
+                if(role && ![role isEqualToString:@"(null)"] && attendeesRoles[role]) {
+                    [formattedAttendee setValue:attendeesRoles[role] forKey:@"role"];
+                }
+                else {
+                    [formattedAttendee setValue:@"Unknown" forKey:@"role"];
+                }
+                if(status && ![status isEqualToString:@"(null)"] && attendeesStatuses[status]) {
```

Comment:
ditto

also, nit: should be a space after if. Interesting, if there is default formatter for objc